### PR TITLE
feature: 1-liner snapshot download/extract cmd

### PR DESCRIPTION
**Pull request by rebelArtists**
**purpose:**
- continue to simplify UX, chaindata snapshots are now a 1-line cli cmd
- add support for split snapshots (backend refactor that split large chaindata files into 25G parts for better caching)
- provided disk type and IOPS guidance to node operators

**tests:**
- fully tested on new nodes
- ensured clients are able to start successfully from snapshots

### CODE REVIEW

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 1]
```diff
@@ -189,3 +242,11 @@ PoS Network is deprecating Archive Node snapshots we request users to move to th
 
 Currently under maintenance. ETA Aug 2023 for Erigon bor-mainnet incremental snapshots.
 
+
+## Recommended disk type and IOPS guidance
+- Disk IOPS will impact speed of downloading/extracting snapshots,
+  getting in sync, and performing LevelDB compaction
+- To minimize disk latency, direct attached storage is ideal
```
Comment (DannyS03) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md:249]:
```suggestion
- To minimize disk latency, direct attached storage is ideal.
```

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 2]
```diff
@@ -189,3 +242,11 @@ PoS Network is deprecating Archive Node snapshots we request users to move to th
...
+- In AWS, when using gp3 disk types, we recommend provisioning IOPS of 16000 and
+  throughput of 1000 - this minimizes cost and adds a lot of performance. io2 EBS volumes with matching IOPS and throughput values are similarly performant
```
Comment (DannyS03) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md:251]:
```suggestion
  throughput of 1000 - this minimizes cost and adds a lot of performance. io2 EBS volumes with matching IOPS and throughput values are similarly performant.
```

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 3]
```diff
@@ -189,3 +242,11 @@ PoS Network is deprecating Archive Node snapshots we request users to move to th
...
+- For GCP, we recommend using performance (SSD) persistent disks (`pd-ssd`) or extreme persistent disks (`pd-extreme`) with similar IOPS and throughput values as seen above
```
Comment (DannyS03) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md:252]:
```suggestion
- For GCP, we recommend using performance (SSD) persistent disks (`pd-ssd`) or extreme persistent disks (`pd-extreme`) with similar IOPS and throughput values as seen above.
```

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 4]
```diff
@@ -189,3 +242,11 @@ PoS Network is deprecating Archive Node snapshots we request users to move to th
 
 Currently under maintenance. ETA Aug 2023 for Erigon bor-mainnet incremental snapshots.
 
+
+## Recommended disk type and IOPS guidance
```
Comment (DannyS03) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md:246]:
```suggestion
## Recommended disk type and IOPS guidance

```

**Review by DannyS03 — APPROVED** [review]
Thank you for the contribution @rebelArtists, lgtm - just some nits and suggestions (not blockers):

- I believe we could just create one function for "joining parts" and then call it like:

```
# Join bulk parts into valid tar.zst and extract
process_parts "$client-$network-snapshot-bulk" 0

# Join incremental following day parts
process_parts "$client-$network-snapshot" 3
```

- We could also generate a markdown table for the "Recommended disk type and IOPS guidance" section instead of a list.

**Comment by rebelArtists** [comment]
agreed, good suggestions, can be a fast follow after rollout

**Pull request merged**